core/arch/arm/plat-imx/a7_plat_init.S

```diff
@@ -34,7 +34,7 @@
 FUNC plat_cpu_reset_early , :
 UNWIND(	.fnstart)
 
-	mov_imm r0, 0x00000040
+	mov_imm r0, 0x00006040
```

Comment:
Nit: I would prefer the usual read modify update cycle.

Reply:
According A7 TRM, only the L1PCTL has a non-zero one. what we do here is just to set SMP bit. I think it is fine to just use mov_imm here.
```
L1 Data prefetch control. The value of this field determines the maximum number of outstanding data
prefetches permitted in the L1 memory system, not counting those generated by software load or PLD
instructions:
0b00 Prefetch disabled.
0b01 1 outstanding pre-fetch permitted.
0b10 2 outstanding pre-fetches permitted.
0b11 3 outstanding pre-fetches permitted, this is the reset value.
```

Reply:
A comment describing this value would be nice.